SCClassLibrary/Platform/linux/LinuxPlatform.sc

```diff
@@ -49,11 +49,13 @@ LinuxPlatform : UnixPlatform {
 			"rxvt -T % -e %",
 			"terminator -T % -e %",
 			"xterm -T % -e %",
+            "kitty -T % -e %",
+            "alacritty -t % -e %",
 			// DE-specific terminals last: avoid problems if not on GNOME or KDE but term installed
 			"xfce4-terminal -T % -e %",
 			"mate-terminal -t % -e %",
 			"gnome-terminal -t % -- %",
-			"konsole --title % -e %",
+			"konsole --title % -e %"
```

Comment:
is this change needed?

Reply:
Hmm, perhaps not - I'll reinstate the comma